guides/source/api_app.md

```diff
@@ -151,6 +151,113 @@ This will do three main things for you:
 - Configure the generators to skip generating views, helpers, and assets when
   you generate a new resource.
 
+### Generating a New Resource
+
+To see how our newly created API handles generating a new resource, let's create
+new Group and Member resources. Each group will have a name, and each member will
+have an associated group_id and name column.
+
+```bash
+$ bin/rails g scaffold Group name:string
+$ bin/rails g model Member name:string group:references
+```
+
+Before we can use our scaffolded code, we need to update our database scheme.
+
+```bash
+$ bin/rails db:migrate
+```
+Our models should be set up as:
+```ruby
+# app/models/group.rb
+class Group < ApplicationRecord
+  has_many :members
+end
+```
+```ruby
+# app/models/member.rb
+class Member < ApplicationRecord
+  belongs_to :group
+end
+```
+
+Now if we open our Groups Controller, we should notice that with an API rails
+app we are rendering JSON data only. On the index action we query for Group.all
+and assign it an instance variable called `@groups`. Passing it to `render` with the
+`:json` option will automatically render the groups as JSON.
```

Comment:
Sorry, still some tiny comments:
```suggestion
Now if we open our `GroupsController`, we should notice that with an API Rails
app we are rendering JSON data only. On the index action we query for `Group.all`
and assign it to an instance variable called `@groups`. Passing it to `render` with the
`:json` option will automatically render the groups as JSON.
```

Reply:
No apologies necessary, thank you for the help! I will try to keep these formatting changes in mind for any future contributions. Still a lot to learn :D